lib/remote/apilistener-configsync.cpp

```diff
@@ -72,6 +72,16 @@ Value ApiListener::ConfigUpdateObjectAPIHandler(const MessageOrigin::Ptr& origin
 		return Empty;
 	}
 
+	String objZone = params->Get("zone");
+
+	if (!ConfigObject::GetObject("zone", objZone)) {
+		Log(LogNotice, "ApiListener")
+				<< "Discarding 'config update object' message"
+				<< " from '" << identity << "' (endpoint: '" << endpoint->GetName() << "', zone: '" << endpointZone->GetName() << "')"
+				<< " for object '" << objName << "' of type '" << objType << "'. Objects zone '" << objZone << "' isn't known locally.";
```

Comment:
_I'd_ not separate two " " by a <<.

Reply:
IMO it's easier to read this way. Other log messages in that function do it the same way.

Reply:
Fine.